openlibrary/templates/type/edition/view.html

```diff
@@ -268,7 +268,11 @@ <h2 class="edition-byline">
       $ seen = set()
       $if previews:
         <p class="preview-languages">
-          Previews available in:
+          $for e in previews:
+            $if len(e.languages)!=0:
+               Previews available in:
```

Comment:
Question, will this result in the words "Previews available in" being displayed multiple times in the page?

Reply:
I see! We have a second for loop and are using `breaks`, that can work!

Reply:
By default, any non-zero number will be a `True` value, so can use `len(e.languages)` as an a equivalent to `len(e.languages)!=0`.

Another option is a list comprehension where we do something like...

`$if previews and any([len(e.languages) for e in previews)]:`

Reply:
Thank you! Do you want me to change it in the way that you suggested?

Reply:
I implemented the changes